install/minikube-installation-guide.md

```diff
@@ -0,0 +1,231 @@
+# Installing Kata Containers in [Minikube](https://kubernetes.io/docs/setup/minikube/)
```

Comment:
Nit: Remove the URL reference in the title. It looks a bit odd to me and that URL is specified in the very first sentence of the document anyway, which I think is conventional.